deploy-review/app.adoc

```diff
@@ -0,0 +1,18 @@
+:gls_prefix:
+
+// Do not modify section headers, as they are used by flamel and the translation process.
+= pass:a,n[{gls_res_outcomes}]
+
+You should be able to deploy Quarkus Conference application, by using different techniques provided by the Red{nbsp}Hat Build of Quarkus.
+
+== Present App FLow
+
+* Logging in to the `Openshift Container Platform` and start using the project created for you.
+* Deploy the `Speaker microservice` using the `Quarkus openshift extension`.
+* Deploy the `Schedule Microservice` using the `Container Image deployment wizard` from the OpenShift Container Platform console.
+
+== Proposed App Flow
+
+* Logging in to the `Openshift Container Platform` and start using the project created for you.
```

Comment:
What does `start using the project` mean? Can we provide a bit more detail?

Reply:
Before deploying the application on Openshift, we login to Openshift and create a project to hold our application with the prefix deploy-review.